client/state/test/initial-state.js

```diff
@@ -10,17 +11,26 @@ import sinon from 'sinon';
 import config from 'config';
 
 describe( 'initial-state', () => {
-	let localforage, createReduxStoreFromPersistedInitialState, MAX_AGE;
+	let localforage, createReduxStoreFromPersistedInitialState, MAX_AGE,
+		isSwitchedUser = false,
+		isSupportUserSession = () => isSwitchedUser;
 
 	require( 'lib/react-test-env-setup' ).auto();
 
 	before( () => {
+		mockery.registerMock( 'lib/user/support-user-interop', { isSupportUserSession: isSupportUserSession } );
+		mockery.enable( { warnOnUnregistered: false } );
 		localforage = require( 'localforage' );
 		const initialState = require( '../initial-state' );
 		createReduxStoreFromPersistedInitialState = initialState.default;
 		MAX_AGE = initialState.MAX_AGE;
 	} );
 
+	after( () => {
+		mockery.deregisterAll();
```

Comment:
It can be simplified now by using `useMockery` helper. See https://github.com/Automattic/wp-calypso/blob/f5bf33b5206c999ff17475900179fb5a846be73e/client/test/use-mockery.js.